src/main/java/com/product/controller/ProductController.java

```diff
@@ -31,25 +31,25 @@ public ProductController(ProductService productService) {
 	}
 
 	@GetMapping("/products")
-	public List<Product> getProduct() {
-		return productService.getAllProducts();
+	public List<Product> getProduct(@RequestParam(name = "order") String order,@RequestParam(name = "containName") String containName) {
```

Comment:
if we dont say `name = ` it should take in the default ? i guess ?

Reply:
Yes , we use name incase we need different name at the client end.